### ISSUE TITLE
Non-motor seizure

### ISSUE THREAD

**Issue by oliviajhenry** [obophenotype/human-phenotype-ontology#5838]
**Preferred term label:**
Non-motor seizure

**Synonyms**
Seizure without motor onset, unclassified seizure without motor onset, unclassified non-motor seizure

**Definition (free text, please give PubMed ID)**
A type of unknown onset seizure characterised by non-motor signs or symptoms (or behaviour arrest) as its initial semiological manifestation. For everyday purposes seizures are broadly categorised as either generalised or focal in onset, these terms can be used where appropriate, but there are seizures that cannot be categorised in this manner and are classified as having unknown onset.
PMID: 28276060

**Parent term (use hpo.jax.org/app)**
Seizure: HP:0001250

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Unknown onset epilepsy

**Your nano-attribution (ORCID)**

**Comment by DLewisSmith** [comment]
This and #5837 are a good suggestions that we considered in the Epilepsiome review. I shall give them some more thought and discuss it within the Task Force. Before discussing this it may be helpful for me to discuss how we came to have the corresponding term for motor seizures.

We decided to have an overarching term for Motor seizure HP:0020219 because:
1) There are various semiologies of motor seizure in which some be difficult to establish if the onset is generalized or focal (Bilateral tonic-clonic, Epileptic spasm, Tonic). Thus we wished to allow these to be coded without having to commit to generalized versus focal onset.
2) Even when the onset is known to be different due to localization by anatomical distribution or EEG, focal and generalized seizure of the same subtype (e.g. myoclonic seizure or tonic seizure) are also phenotypically similar because of a shared characteristic (e.g. myoclonic jerking or tonic contraction) when considered independently of anatomical distribution. Thus there may be overlap in the underlying mechanisms of these subtypes across the generalized and focal dichotomy and we wished to be able to identify individuals who may share genetic variants responsible for these shared features.

Thus, we had a number of motor seizure concepts which are independent of onset. These have more in common with each other (they involve the motor cortex) than with non-motor forms of seizure so it made sense to group them. Additionally this may lead to benefits through fuzzy logic: some 'vibratory' tonic and tonic-clonic seizures may be misclassified, as can clonic and myoclonic seizures.

Deciding whether all seizures that are not motor should be grouped under an umbrella term is not quite so straightforward. The semiologies (symptoms or occasionally non-motor signs) of focal seizures are not usually confusable or similar to generalized non-motor seizures. The common exceptions are those instances of seizures characterised predominantly by behavioural arrest and impairment of awareness without other clinical or EEG evidence to distinguish the onset. These instances could be a Focal impaired awareness behavior arrest seizure HP:0032790 or a Generalized non-motor (absence) seizure HP:0002121. We proposed to retain the term Dialeptic seizure HP:0011146 as a parent for both of these terms that is by definition independent of electrographic onset. These relationships are currently not represented within the live HPO version but @pnrobinson and I have been trying to correct the last few errors from the Epilepsiome revision upload. However, the term Dialeptic (created by Luders et al. at the Cleveland Clinic) is controversial amongst some and not in the new ILAE Classification (which in itself has not received global concordance). Some people interpret Dialeptic more broadly than we have suggested (they might suggest that it should be the parent of Focal non-motor impaired awareness seizure HP:0032716 and Generalized non-motor (absence) seizure HP:0002121). However, we were keen to make our seizure subontology bilingual to classification and to include concepts that the most expert epileptologists consider valuable, both ILAE-endorsed and not.

I'm keen to hear more of your thoughts and comments on this @oliviajhenry

**Comment by DLewisSmith** [comment]
@pnrobinson I think we should include 'Behavioral arrest seizure irrespective of onset' and if acceptable 'Unknown onset behavioral arrest seizure' as synonyms of 'Dialeptic seizure' HP:0011146. I accept that the second synonym would not follow the logic of a full synonym (as its child terms would have known onsets) but this is the term used by the ILAE Classification and non-ontologists are likely to search by it and expect it to be represented. However, I would be careful about using the wording 'Behavioral arrest seizure' as the preferred term or synonym for the current concept of 'Diaelptic seizure' without explicitly specifying that this is independent of onset because some people may mistakingly assume that 'Behavioral arrest seizure' is focal in onset.

'Focal impaired awareness behavior arrest seizure' HP:0032790 should be a child of 'Dialeptic seizure' HP:0011146 as well as being a child of 'Focal behavior arrest seizure' HP:0011173.

**Comment by oliviajhenry** [comment]
Hi @DLewisSmith. Apologies for the delayed response, I wished to discuss this with colleagues which took a few weeks due to the summer break. In regards to your prior comment, it is true that the term dialeptic is controversial amongst some. Beyond some people interpreting the term ‘dialeptic’ more broadly (as you noted that some may suggest it should be the parent of a focal non-motor impaired awareness seizure and generalised non-motor (absence) seizure), there are other instances where there may be confusion around its usage. Blumenfeld & Meador (Epilepsia, 2014) point out that while there are similarities in the presentation of generalised non-motor (absence) seizures and focal impaired awareness behavioural arrest seizures there are important differences in the pathophysiology, behaviour, treatment, outcome and patient populations. However if you/ the Task Force you believe dialeptic represents the term which is most frequently used by the community (and acknowledging the difficulty of allocating a an apt term to this particular group) then listing ‘unknown onset behavioural arrest seizure’ as a synonym should be sufficient.  Indeed, explicitly stating the unknown onset seems like a wise amendment to avoid confusion with focal onset behavioural arrest seizures.
To avoid the issue you pointed out of offspring of ‘unknown onset behavioural arrest seizure’ the structure could instead be as follows:
Parent term: dialeptic/ behavioural arrest seizure irrespective of onset
Offspring: unknown onset behavioural arrest seizure, focal impaired awareness behavioural arrest seizure (HP:0032790) and generalised non-motor (absence) seizure (HP:0002121).

**Comment by DLewisSmith** [comment]
Thanks @oliviajhenry, thanks for getting back to us. I entirely agree with the importance of differences between Absences and FIAS being recognised, this is the reason for keeping them as separate concepts, which if coded explicitly allow generalized vs focal seizure types to be inferred. After all, this subontology is based on the ILAE Fisher Classification, and thus, we aim to classify seizures where diagnostically possible into Focal vs Generalized onset. The relationships between non-motor seizures in an ontology are issues I have been wrestling with for some time, particularly because in my clinical practice with people who have severe/profound intellectual disability and epilepsy we frequently have to accept that a seizure is characterised by behavioural arrest and is either a form of absence or FIA non-motor seizure but a prolonged EEG is unlikely to be tolerated, and we cannot reach the 80% threshold to declare the seizure as focal vs generalised.

Whilst comfortable to classify in the ILAE Fisher Classification (if we accept that a person with profound ID may have their Focal IA sensory seizure classified as a Focal behavioural arrest seizure as a consequence of not being able to report their visual aura), this raises various challenges for an ontology in which we aim to allow the maximum amount of information to be coded about any particular instance of a seizure, and in which we use ontology structure to handle uncertainty. We need a concept that represents those seizures with behavioural arrest without sufficiently distinguishing features to code them as another seizure type, or even of being of focal or generalised onset. Whilst this concept may be a second parent of absences, it cannot be said to be a parent of all forms of Focal IA non-motor seizure, because in many (if not most) cases a Focal IA non-motor seizure can be confidently diagnosed based on its electroclinical features. We stuck with 'Dialeptic seizure' as the primary name because it existed prior to our revision, its definition seems appropriate, and it remains in use amongst many neurologists. We would hope that definite FIA non-motor seizures would be coded as such given the range of options available from the ILAE Fisher Classification rather than as a 'Dialeptic seizure' (for example, people should code a 'Focal impaired awareness seizure' as such rather than a 'Dialeptic seizure'; additionally there are many more specific forms of FIAS in the subontology). However, as you'll have noticed we've been careful with the primary names we've picked for other concepts such as GTCS vs FBTC because there are consultant epileptologists who use 'GTCS' indiscriminately.

It seems that we agree that ‘unknown onset behavioural arrest seizure’ should be a synonym of 'Dialeptic seizure' (HP:0011146) if acceptable to @pnrobinson. However, I do not think that a 'unknown onset behavioural arrest seizure' should also be a child of 'Dialeptic' as an the concept is distinguished from 'Focal impaired awareness behavioural arrest seizure' (HP:0032790) and 'Generalised non-motor (absence) seizure' (HP:0002121) by being of 'unknown onset' which is a property of our diagnostic limitations rather than an intrinsic property of the seizure itself - thus an 'unknown onset behavioural arrest seizure' should be coded as a 'Dialeptic seizure' which cannot be coded more specifically as either a 'Focal impaired awareness behavioural arrest seizure' (HP:0032790) and 'Generalised non-motor (absence) seizure' (HP:0002121) at that point in time.

**@pnrobinson I think this is where we stand:
1) 'Focal impaired awareness behavior arrest seizure' HP:0032790 should be a child of 'Dialeptic seizure' HP:0011146 as well as being a child of 'Focal behavior arrest seizure' HP:0011173.
2) ‘Unknown onset behavioural arrest seizure’ (this is because users will tend to search for classification terms in the ontology) and 'Behavioral arrest seizure irrespective of onset' should be synonyms of 'Dialeptic seizure' (HP:0011146) if acceptable**

I think this is an area on which we are likely to need to review based on user feedback. Thanks @oliviajhenry

**Comment by pnrobinson** [comment]
@DLewisSmith  This is the current version for Focal impaired awareness behavior arrest seizure' -- 3 parents. Is this OK?
![image](https://user-images.githubusercontent.com/4943420/100518254-77b57680-315e-11eb-9ffe-679eb64bc6df.png)

**Comment by pnrobinson** [comment]
Additionally, I added Unknown onset behavioural arrest seizure and Behavioral arrest seizure irrespective of onset' as synonyms of 'Dialeptic seizure

Please close this item if this was the correct action.

**Comment by DLewisSmith** [comment]
@pnrobinson sorry I think following our consideration about the harmonization of the two classifications (particularly the seizure types vs individual seizure components) it would be best to leave “Dialeptic seizure” exactly as it is in the current release with additional PMID. This interprets "Dialeptic seizure" more broadly, meaning that coding of any seizure with a dialeptic component under this term would be correct, where as our previous suggestion really only applies to the concept of a seizure (with only a single component, which is dialeptic). This arrangement is more consistent with current use and PMID:30782582 & 28276060.

I will post a separate a suggestion below for the concept we previously tried to capture here, which is moving back closer to the suggestion by @oliviajhenry

Definition: as in current release
PMID: 30782582, 9738682, 
-1x parent:
"Seizure" HP:0001250
-2x children:
“Generalized non-motor (absence) seizure” HP:0002121
“Focal impaired awareness seizure” HP:0002384

**Comment by DLewisSmith** [comment]
@pnrobinson @oliviajhenry How does this seem to you - it's closer to Olivia's suggestion (and ironically where we stood in Luxembroug)? Any suggested modifications?

After further consideration of how to best harmonise ILAE and 4-dimension concepts (regardless of electrographic onset) it would be helpful to separate Dialeptic and Non-motor characteristics.

New term: "Non-motor seizure"
Definition: "A seizure with clinical manifestation but without motor signs (other than possible behavior arrest) as its initial clinical manifestation. The electrographic onset may be generalized, focal, or unknown."
Comment: “This term may be used to code the non-motor seizures of a person who cannot confirm or exclude subjective seizure experiences that would aid more detailed classification as a generalized non-motor (absence) seizure or type of focal non-motor seizure.”
PMID: 28276060, 28276064

-1x parent:
"Seizure" HP:0001250
-2x children:
“Generalized non-motor (absence) seizure” HP:0002121
“Focal non-motor seizure” HP:0032679

**Comment by pnrobinson** [comment]
done
